src/components/UI/LoadingOverlayWithoutStyles.js

```diff
@@ -0,0 +1,27 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+// LoadingOverlay takes one boolean prop 'loading' and will render a
+// page centered loading spinner if 'loading' is true, or its children if
+// 'loading' is false.
+//
+// Usage:
+//
+// <LoadingOverlay loading={something.hasLoaded}>
+//    <h1>{something.that.only.is.here.when.fully.loaded}</h1>
+// </LoadingOverlay>
+
+function LoadingOverlayWithoutStyles(props) {
+  if (props.loading) {
+    return <img className='loader' src='/images/loader_oval_light.svg' />;
```

Comment:
Can you please import the image as a module, [as specified in fmt](https://github.com/giantswarm/fmt/blob/master/javascript/general_style_guide.md) ?